client/src/components/DogParksList.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from "react"
+import { Link } from "react-router-dom"
+
+const DogParksList = props => {
+    const [ dogParks, setDogParks ] = useState([])
+
+    const getParks = async () => {
+        try {
+            const response = await fetch("/api/v1/parks")
+            const responseBody = await response.json()
+            if (!response.ok) {
+                throw new Error(`${response.status} (${response.statusText})`)
+            }
+            setDogParks(responseBody.parks)
+        } catch(err) {
+            console.error("Error in fetch",err.message)
+            res.set(500).json( {errors: err.data} )
```

Comment:
wohhh, this syntax on line 17 is for express, and would very much break your app. Please remove